core-java-modules/core-java-lang-oop-types-2/src/main/java/com/baeldung/enumtostring/FastFood1.java

```diff
@@ -0,0 +1,25 @@
+package com.baeldung.enumtostring;
+
+public enum FastFood1 {
+    PIZZA,
+    BURGER,
+    TACO,
+    CHICKEN,
+    ;
+
+    @Override
+    public String toString() {
+        switch (this) {
+        case PIZZA:
+            return "Pizza Pie";
+        case BURGER:
+            return "Cheese Burger";
+        case TACO:
+            return "Crunchy Taco";
+        case CHICKEN:
+            return "Fried Chicken";
+        default:
+            return null;
```

Comment:
In this cases, it's better to return an empty String.

Reply:
Done